<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Encrypt File</string>
     </attribute>
     <widget class="QLabel" name="file_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>File</string>
      </property>
     </widget>
     <widget class="QLabel" name="output_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Location</string>
      </property>
     </widget>
     <widget class="QLabel" name="recipient_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirIn_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirOut_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recipient_lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>281</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="input_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QPushButton" name="output_button">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encrypt_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decrypt_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Email File</string>
     </attribute>
     <widget class="QLabel" name="file_label2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailDir_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="input_button2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostname_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hostname</string>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostname_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="email_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="recipient_label2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recipient_lineEdit2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>311</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipient</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>recipient_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>recipient_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirOut_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dirOut_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirIn_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>dirIn_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_button</sender>
   <signal>clicked()</signal>
   <receiver>dirIn_lineEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_button</sender>
   <signal>clicked()</signal>
   <receiver>dirOut_lineEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_button2</sender>
   <signal>clicked()</signal>
   <receiver>emailDir_lineEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostname_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>hostname_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>password_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>username_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>username_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recipient_lineEdit2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>recipient_lineEdit2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailDir_lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>emailDir_lineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
